### DISCUSSION TITLE
页眉相关问题

### DISCUSSION THREAD

**Discussion by Aphroditefang** [stone-zeng/fduthesis#262] [Q&A] [answered]
您好，关于页眉有两个问题：
1.
![image](https://user-images.githubusercontent.com/34327524/175452707-41630847-1d47-4825-9484-a27ac7af3c82.png)
目录这边的页眉怎么去掉呢？

2.
![image](https://user-images.githubusercontent.com/34327524/175452972-412e21a4-75e7-46bf-856a-9e2db6389b47.png)
这里参考文献能不能移到右边去呢？

**Comment by stone-zeng** [comment, marked as answer]
See #263.
